web/gui/dashboard_info.js

```diff
@@ -1171,6 +1171,19 @@ netdataDashboard.context = {
         info: 'Bytes received by functions <code>tcp_cleanup_rbuf</code> and <code>udp_recvmsg</code>.'
     },
 
+    'apps.bandwidth_tcp_send': {
+        info: 'Calls for function <code>tcp_sendmsg</code>.'
+    },
+
+    'apps.bandwidth_tcp_recv': {
+        info: 'Calls for functions <code>tcp_cleanup_rbuf</code>.'
+    },
+
+    'apps.bandwidth_tcp_retransmit': {
+        info: 'Calls for functions <code>tcp_retranstmit_skb</code>.'
+    },
```

Comment:
should we add info for `udp_sendmsq` and `udp_recvmsg`?

Reply:
I missed them, thank you!